simulation-construction-set-examples/falling-brick/src/main/java/us/ihmc/fallingBrick/ClutteredGroundDefinition.java

```diff
@@ -0,0 +1,73 @@
+package us.ihmc.fallingBrick;
+
+import us.ihmc.euclid.transform.RigidBodyTransform;
+import us.ihmc.scs2.definition.collision.CollisionShapeDefinition;
+import us.ihmc.scs2.definition.geometry.Box3DDefinition;
+import us.ihmc.scs2.definition.geometry.Cylinder3DDefinition;
+import us.ihmc.scs2.definition.geometry.GeometryDefinition;
+import us.ihmc.scs2.definition.geometry.PyramidBox3DDefinition;
+import us.ihmc.scs2.definition.geometry.Sphere3DDefinition;
+import us.ihmc.scs2.definition.terrain.TerrainObjectDefinition;
+import us.ihmc.scs2.definition.visual.ColorDefinitions;
+import us.ihmc.scs2.definition.visual.MaterialDefinition;
+import us.ihmc.scs2.definition.visual.VisualDefinition;
+
+public class ClutteredGroundDefinition extends TerrainObjectDefinition
```

Comment:
Let's add some doc here:
The terrain object definition is used to visualize a static environment and to define its collision model. The collision model is used to resolve contact between the robots and the environment.

Reply:
Added